src/assets/i18n/en.json5

```diff
@@ -2298,6 +2305,22 @@
 
   "item.page.claim.tooltip": "Claim this item as profile",
 
+  "item.page.subscriptions.tooltip": "Subscribe",
```

Comment:
I find it odd that many of the subscription i18n tags start with "item.page" when they are obviously used for *Communities, Collections and Items*.  These same keys are used for all DSOs.

Maybe these should just be named `subscriptions.tooltip", etc (drop the entire "item.page" par, and move these down with the other `subscriptions.*` keys)?

I don't think these should start with "item.page" as that implies these are only used for items